specs/darwin/crash_logs.table

```diff
@@ -0,0 +1,27 @@
+table_name("crash_logs")
+description("Application Crash Logs")
+schema([
+    Column("process", TEXT, "Name of the process that crashed."),
+    Column("application_path", TEXT, "Path to the crashed process."),
+    Column("log_location", TEXT, "Location of log file."),
+    Column("identifier", TEXT, "Identifier of the crashed process."),
+    Column("version", TEXT, "Version info of the crashed process."),
+    Column("parent_proc", TEXT, "Parent of the crashed process."),
```

Comment:
Similar to the process note, the value in the logs is the name of the process followed by a PID.  I can, as with the Process Name, split this out into the parent PID and the parent proc name if we want?